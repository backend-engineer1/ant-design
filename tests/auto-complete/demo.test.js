import demoTest from '../shared/demoTest';

demoTest('auto-complete');
