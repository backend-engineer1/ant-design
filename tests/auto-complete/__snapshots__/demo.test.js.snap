exports[`test renders ./components/auto-complete/demo/basic.md correctly 1`] = `
<div
  class="ant-select-show-search ant-select ant-select-combobox ant-select-enabled"
  style="width:200px;">
  <div
    aria-autocomplete="list"
    aria-expanded="false"
    aria-haspopup="true"
    class="ant-select-selection
              ant-select-selection--single"
    role="combobox">
    <div
      class="ant-select-selection__rendered">
      <div
        class="ant-select-selection__placeholder"
        style="display:block;user-select:none;-webkit-user-select:none;"
        unselectable="unselectable">
        input here
      </div>
      <ul>
        <li
          class="ant-select-search ant-select-search--inline">
          <div
            class="ant-select-search__field__wrap">
            <input
              class="ant-select-search__field"
              value="" />
            <span
              class="ant-select-search__field__mirror" />
          </div>
        </li>
      </ul>
    </div>
    <span
      class="ant-select-arrow"
      style="user-select:none;-webkit-user-select:none;"
      unselectable="unselectable">
      <b />
    </span>
  </div>
</div>
`;

exports[`test renders ./components/auto-complete/demo/options.md correctly 1`] = `
<div
  class="ant-select-show-search ant-select ant-select-combobox ant-select-enabled"
  style="width:200px;">
  <div
    aria-autocomplete="list"
    aria-expanded="false"
    aria-haspopup="true"
    class="ant-select-selection
              ant-select-selection--single"
    role="combobox">
    <div
      class="ant-select-selection__rendered">
      <div
        class="ant-select-selection__placeholder"
        style="display:block;user-select:none;-webkit-user-select:none;"
        unselectable="unselectable">
        input here
      </div>
      <ul>
        <li
          class="ant-select-search ant-select-search--inline">
          <div
            class="ant-select-search__field__wrap">
            <input
              class="ant-select-search__field"
              value="" />
            <span
              class="ant-select-search__field__mirror" />
          </div>
        </li>
      </ul>
    </div>
    <span
      class="ant-select-arrow"
      style="user-select:none;-webkit-user-select:none;"
      unselectable="unselectable">
      <b />
    </span>
  </div>
</div>
`;
